Fix building with xulrunner-1.9.2 by including nspr CFLAGS as well

http://bugs.gentoo.org/296924

---
--- a/gtkmozembed/Makefile.am
+++ b/gtkmozembed/Makefile.am
@@ -1,7 +1,7 @@
 # require automake-1.5
 AUTOMAKE_OPTIONS = 1.5
 
-INCLUDES = $(PYTHON_INCLUDES) $(PYGTK_CFLAGS) $(GTKMOZEMBED_CFLAGS)
+INCLUDES = $(PYTHON_INCLUDES) $(PYGTK_CFLAGS) $(GTKMOZEMBED_CFLAGS) $(NSPR_CFLAGS)
 
 defsdir = $(datadir)/pygtk/2.0/defs
 defs_DATA = gtkmozembed.defs
--- a/configure.ac
+++ b/configure.ac
@@ -24,6 +24,7 @@
 m4_define(gksu_required_version,              1.2.5)
 m4_define(gksu2_required_version,             2.0.4)
 m4_define(gksuui_required_version,            1.0.3)
+m4_define(nspr_required_version,	4.7.4)
 
 AC_INIT(gnome-python-extras, gnome_python_extras_version,
         [http://bugzilla.gnome.org/enter_bug.cgi?product=gnome-python])
@@ -62,6 +63,11 @@
 AM_PATH_PYTHON(2.3)
 AM_CHECK_PYTHON_HEADERS(,[AC_MSG_ERROR(could not find Python headers)])
 
+dnl check for nspr
+AC_MSG_CHECKING(for system nspr >= nspr_required_version )
+PKG_CHECK_MODULES(NSPR, nspr >= nspr_required_version)
+AC_SUBST(NSPR_CFLAGS)
+
 dnl check for pygtk
 PKG_CHECK_MODULES(PYGTK, pygtk-2.0 >= pygtk_required_version)
 AC_SUBST(PYGTK_CFLAGS)

